wiki/Contests/osb!_Storyboarding_Contest/1/en.md

```diff
@@ -0,0 +1,111 @@
+| ![][flag_VN] [Hokichi](https://osu.ppy.sh/users/6607303) | Co-host, graphic designer, judge |
+| ![][flag_GB] [WildOne94](https://osu.ppy.sh/users/3482692) | Staff member, judge |
+| ![][flag_AU] [Coppertine](https://osu.ppy.sh/users/7279762) | Staff member, judge |
+| ![][flag_DE] [Darky1](https://osu.ppy.sh/users/3372928) | Staff member, judge |
+| ![][flag_CA] [Marblelemons](https://osu.ppy.sh/users/8286416) | Staff member, judge |
+| ![][flag_BR] [Dada](https://osu.ppy.sh/users/9119507) | Judge |
+| ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | Judge |
+| ![][flag_US] [Starrodkirby86](https://osu.ppy.sh/users/410) | Judge |
+| ![][flag_SE] [Naxess](https://osu.ppy.sh/users/8129817) | Logo designer |
+
+## Links
+
+- **[Discussion thread](https://osu.ppy.sh/community/forums/topics/897075)**
+- [Judging spreadsheet](https://docs.google.com/spreadsheets/d/16hHsR_foAqDAqckcZ33Om-OLxqgtNXzFqWqDSe7gdnU/edit#gid=0)
+- [1st place storyboard showcase](https://www.youtube.com/watch?v=dFfXZfOJQPs)
+- [2nd place storyboard showcase](https://www.youtube.com/watch?v=ucRjWfP6ZSA)
+- [3rd place storyboard showcase](https://www.youtube.com/watch?v=F4QL4B4Qgso)
+
+## Participants
+
+| Storyboarder | Entry |
+| :-- | :-- |
+| ![][flag_CA] [Gaia](https://osu.ppy.sh/users/2683648) | [puru - Grimheart (Original Mix & Cut Ver.) \[RED FOX\]](https://drive.google.com/uc?id=1dIBPlOdB00Mu-To8SdAwgGsi9OUSQ1JT) |
+| ![][flag_UA] [PantyDev](https://osu.ppy.sh/users/5420543) | [Itou Kanako - COSMIC LOOPER (TV Size) \[MAINE COON\]](https://drive.google.com/uc?id=12i-RTXTOM60P1MfBx_QGBMwmB5qOWXbz) |
+| ![][flag_CA] [TheDuckMask](https://osu.ppy.sh/users/7405768) | [AVTechNO!xTreow - DYE/Re:flection+ \[SHIBA INU\]](https://drive.google.com/uc?id=1y51gyjGgrrIQJlxkjxNoGRwFu285KaeH) |
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Place | User |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_CA] [TheDuckMask](https://osu.ppy.sh/users/7405768) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_UA] [PantyDev](https://osu.ppy.sh/users/5420543) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_CA] [Gaia](https://osu.ppy.sh/users/2683648) |
+
+## Ruleset
+
+### General rules
+
+- **You can choose any song, theme and sprites (including your own)**. This is so that we do not limit your potential and idea.
+- **Mapping is not required**. This is a storyboarding contest!
+- Use of **provided sprites are optional**, but it is **strongly encouraged since points are calculated with sprites and their usage**.
+- If you get a **name change** during the duration of the tournament, please **let hosts know as soon as possible**!
+- **No previous experience necessary**. (This includes osu!editor’s design tab, programming and photoshop, but would be a great advantage)
```

Comment:
```suggestion
- **No previous experience necessary**. (This includes experience with osu!editor's design tab, programming and photoshop, although it would be a great advantage.)
```